server/src/config.js

```diff
@@ -13,10 +13,17 @@ let conf = convict({
     env: "PORT",
     arg: "port"
   },
+  host: {
+    doc: "The server host",
+    format: String,
+    default: "localhost",
+    env: "HOST",
+    arg: "host"
+  },
   contentPort: {
```

Comment:
Yeah, on reflection I think it's fine, the extra configuration isn't a big deal.